Backend/ResourceServer/ResourceServer/Controllers/VisitorController.cs

```diff
@@ -24,13 +24,19 @@ public async Task<ActionResult<Visitor>> CreateVisitor([FromBody] VisitorDTO dto
         {
 
             var createdVisitor = await _visitorRepository.CreateVisitor(dto);
+
             var visitorAccount = await _visitorAccountRepository.GetVisitorAccountById(dto.VisitorAccountId);
 
             if (visitorAccount == null)
             {
                 return NotFound();
             }
 
+            if (createdVisitor == null)
+            {
+                return BadRequest("Country not found");
```

Comment:
Country = visitor?

Reply:
Tänkte eftersom du hade lagt till om den inte hittar country så retunerar du null så la bara till den så man får en idikation på att det var country man hade skrivit fel, men vi kan ändra på den om det inte var bra

Reply:
On the question on adding to.lower in auth service i though that in the frontend they could lower the username being sent instead of lowering it in the backend but we could do it in the backend as well

Reply:
Ok, no country is correct, I was looking at an old branch.

Reply:
I think adding to lower case in the backend is a good extra layer of validation